charts/nexus-iq/README.md

```diff
@@ -4,19 +4,21 @@
 
 ### Prerequisites
 
-- Kubernetes 1.8+ with Beta APIs enabled
+- Kubernetes 1.19+
```

Comment:
They're up to k8s 1.23.0, 1.22 removed support for beta ingress in favor of full v1.